src/main/java/org/apache/accumulo/testing/cli/ClientOpts.java

```diff
@@ -169,34 +165,20 @@ public String getClientConfigFile() {
     return clientConfigFile;
   }
 
-  public Properties getClientProperties() {
+  public Properties getClientProps() {
     if (cachedProps == null) {
       cachedProps = new Properties();
       if (getClientConfigFile() != null) {
         cachedProps = toProperties(Paths.get(getClientConfigFile()));
       }
-      if (saslEnabled) {
-        cachedProps.setProperty(ClientProperty.SASL_ENABLED.getKey(), "true");
-      }
-      if (sslEnabled) {
-        cachedProps.setProperty(ClientProperty.SSL_ENABLED.getKey(), "true");
-      }
       if (principal != null) {
         cachedProps.setProperty(ClientProperty.AUTH_PRINCIPAL.getKey(), principal);
       }
-      if (zookeepers != null) {
-        cachedProps.setProperty(ClientProperty.INSTANCE_ZOOKEEPERS.getKey(), zookeepers);
-      }
-      if (instance != null) {
-        cachedProps.setProperty(ClientProperty.INSTANCE_NAME.getKey(), instance);
-      }
       if (securePassword != null) {
         ClientProperty.setPassword(cachedProps, securePassword.toString());
-      } else if (password != null) {
-        ClientProperty.setPassword(cachedProps, password.toString());
-      } else if (keytabPath != null) {
-        ClientProperty.setKerberosKeytab(cachedProps, keytabPath);
       }
+      getOverrides().forEach((k, v) -> cachedProps.put(k, v));
```

Comment:
I wonder if `cachedProps::put` works instead of explicitly putting in the params.

Reply:
Yes it works.